cached_proc_macro/src/cached.rs

```diff
@@ -255,7 +259,7 @@ pub fn cached(args: TokenStream, input: TokenStream) -> TokenStream {
             let old_val = {
                 #lock
                 let (result, has_expired) = cache.cache_get_expired(&key);
-                if let (Some(result), false) = (result, has_expired) {
+                if let (Some(result), false) = (result.clone(), has_expired) {
```

Comment:
hey @Raz-Hemo (sorry for the delay) - doesn't the `#return_cache_block` immediately clone result again? I think it's the flavor defined on line 171

Reply:
no pressure :) you are correct, i changed it now to take result by ref here instead of cloning